docs/faq/nuclei.md

```diff
@@ -21,29 +21,29 @@
 
 ??? info "What kind of scans can I perform with nuclei?"
 
-	Nuclei can be used to detect security vulnerabilities in **Web Applications**, **Networks**, **DNS** based misconfiguration, and **Secret scanning** in source code or files on the local file system.
+	Nuclei can be used to detect security vulnerabilities in **Web Applications**, **Networks**, **DNS** based misconfiguration, and **Secrets scanning** in source code or files on the local file system.
 
 ??? info "How well-maintained is this project?"
 
-	Nuclei project is developed and maintained by [ProjectDiscovery](https://projectdiscovery.io/#/) team and is in active development (we tend to push release every other week).
+	The Nuclei project is developed and maintained by [ProjectDiscovery](https://projectdiscovery.io/#/) team and is in active development (we generally release every other week).
 
 ??? tip "How can I support/contribute to this project? 💙"
 
-	To keep us motivated to work on this project, we request you to write/share new nuclei templates with the community at [template project](https://github.com/projectdiscovery/nuclei-templates) and help us to maintain this public and ready to use / up-to-date nuclei templates.
+	To keep us motivated to work on this project, we request you to write/share new nuclei templates with the community at [template project](https://github.com/projectdiscovery/nuclei-templates) and help us to maintain this public and ready to use, up-to-date nuclei templates.
```

Comment:
"write/share new nuclei templates with the community...public ready to use, up-to-date nuclei templates". There are a few repetitions in this sentence that seems to be a bit too much.